test/lib/ufe/testSelection.py

```diff
@@ -445,5 +445,53 @@ def testMayaSelectSwitchVariant(self):
 
         self.assertTrue(sn.empty())
 
+    @unittest.skipUnless(mayaUtils.previewReleaseVersion() >= 125, 'Requires Maya fixes only available in Maya Preview Release 125 or later.')
```

Comment:
Shouldn't this be 126 or later?  Customers will have Preview Release 125, and it doesn't have your fix.